integration/spark/src/main/scala/org/apache/spark/sql/test/util/QueryTest.scala

```diff
@@ -207,6 +208,45 @@ class QueryTest extends PlanTest {
       }
     }
   }
+
+  def getRedirectCsvPath(dbName: String,
```

Comment:
may be add all these bad record methods to a scala object specific to bad record util functions and use in all places

Reply:
done